deploy/charts/hcloud-csi/example-prod.values.yaml

```diff
@@ -0,0 +1,97 @@
+fullnameOverride: "csi"
```

Comment:
I don't think we need this example values file. Anything "production specific" in here should just be part of the defaults in `values.yaml`.

Or to put it differently: the default deployment of csi-driver should indeed be a production deployment :)